x-pack/test_serverless/api_integration/test_suites/security/platform_security/authorization.ts

```diff
@@ -265,6 +265,7 @@ export default function ({ getService }: FtrProviderContext) {
                 "api:bulkGetUserProfiles",
                 "api:securitySolution-entity-analytics",
                 "api:securitySolution-threat-intelligence",
+                "api:securitySolution-writeGlobalArtifacts",
```

Comment:
in this file it is visible, that `api:securitySolution-writeGlobalArtifacts` has 2 more occurrences than `ui:siemV3/writeGlobalArtifacts`, namely, the extra occurrences are in `siem.minimal_all` and `siemV2.minimal_all.
the reason is in this comment: https://github.com/elastic/kibana/pull/225678#discussion_r2172402516